docs/user/customization.md

```diff
@@ -12,14 +12,53 @@ The most simple customization is exposed by the installer as an interactive seri
 
 While the default cluster size may be sufficient for some, many will need to make alterations. This can include increasing the number of machines in the control plane, changing the type of the virtual machines that will be used (e.g. AWS instances), or adjusting the CIDR range used for the Kubernetes service network. This level of customization is exposed via the installer's `install-config.yaml`. The install-config can be accessed by running `openshift-install create install-config`. This file can then be modified as needed before running a later target.
 
-The `install-config.yaml` generated by the installer will not have all of the available fields populated, so they will need to be manually added if they are needed. The full list of available fields can be found in the [Go Docs][godocs]. Documentation for each of the supported platforms can be found in their platform-specific section:
+The `install-config.yaml` generated by the installer will not have all of the available fields populated, so they may need to be manually added if they are needed.
+
+### Platform-agnostic properties
+
+The following `install-config.yaml` properties are supported on all platforms:
```

Comment:
Missing `metadata.name` - the name of the cluster

Reply:
> Missing `metadata.name` - the name of the cluster

Lots of stuff still missing ;).  But fixed this particular thing in 2c86ce813 -> 96b16e9d4.